monitoring/api/v3/requirements.txt

```diff
@@ -1,10 +0,0 @@
-google-api-python-client==1.6.2
```

Comment:
This is old file, new one is in api-client, Github just confused with the changes